packages/palette/package.json

```diff
@@ -60,12 +60,12 @@
     "@babel/preset-env": "7.4.4",
     "@babel/preset-react": "7.0.0",
     "@babel/preset-typescript": "7.3.3",
-    "@storybook/addon-actions": "5.0.11",
-    "@storybook/addon-knobs": "5.0.11",
-    "@storybook/addon-links": "5.0.11",
-    "@storybook/addon-viewport": "5.0.11",
-    "@storybook/addons": "5.0.11",
-    "@storybook/react": "5.0.11",
+    "@storybook/addon-actions": "5.3.19",
+    "@storybook/addon-knobs": "5.3.19",
+    "@storybook/addon-links": "5.3.19",
+    "@storybook/addon-viewport": "5.3.19",
+    "@storybook/addons": "5.3.19",
+    "@storybook/react": "5.3.19",
```

Comment:
Wanted to update these because the viewports addon was sort of broken in the older versions. I'm not seeing the route reloading issue we saw in Reaction.

Reply:
🤔